src/modules/common/Loader.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { FC } from 'react';
+import { FC } from 'react';
 
-import { Box } from '@mui/material';
+import { Stack } from '@mui/material';
 import CircularProgress from '@mui/material/CircularProgress';
 
 const Loader: FC = () => (
-  <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+  <Stack display="flex" justifyContent="center">
```

Comment:
`display="flex"` is implicit for `Stack`
```suggestion
  <Stack justifyContent="center">
```